app/views/site/study.html.erb

```diff
@@ -4,10 +4,10 @@
 <div id="tab-root">
   <%= render :partial => 'study_tabs_nav' %>
   <div class="tab-content top-pad">
-    <div class="tab-pane active in" id="study-summary" role="tabpanel">
+    <div class="tab-pane <%= @explore_tab_default ? nil : ' active in' %>" id="study-summary" role="tabpanel">
       <%= render partial: 'study_description_view' %>
     </div>
-    <div class="tab-pane" id="study-visualize" role="tabpanel">
+    <div class="tab-pane <%= @explore_tab_default ? 'active in' : nil %>" id="study-visualize" role="tabpanel">
```

Comment:
Can you remind me how these tabs from the tabs in `_study_tabs_nav.html.erb`?  Are both related to the Explore tab in Study Overview, or is there an Explore tab outside Study Overview that I'm forgetting?

Reply:
Yes, it is a little confusing - I had to remind myself while doing this.  The `_study_tabs_nav.html.erb` partial is just the navigation portion of the Study Overview page.  These `div` elements here are the content panes that get shown/hidden when the nav items are clicked.